compiler-plugin-test/src/test/resources/kotlin/IT_EXT_ALGN_for_exampleAggregate.kt

```diff
@@ -0,0 +1,15 @@
+import it.unibo.collektive.aggregate.api.Aggregate
+
+fun Aggregate<Int>.exampleAggregate() {}
+
+// ========= ========= ========= ========= ========= ========= ========= ========= ========= =========
+// END OF BOILERPLATE CODE
+// ========= ========= ========= ========= ========= ========= ========= ========= ========= =========
```

Comment:
what's this?

Reply:
Since these resources files have been auto-generated (like I described in [this comment](https://github.com/Collektive/collektive/pull/508#issuecomment-2424967681)) I've left a comment on each generated file to mark the end of the common code between those sources. It's just an easy way to see where the real tested code begins.

Should I delete it?

Reply:
Maybe make it a bit more explanatory:

```
// Code from here has been generated, see <link>
...
...
// End of generated code. Core testing begins here.
```

Reply:
Actually, all the code inside these resource files is generated, the part before the comment is just identical between all tested sources, so it's not really meaningful to the test. 

But I see your point: I will put a comment at the start indicating that the code inside is all generated. The code to generate these sources is not online yet, maybe I should put it in my fork and add a link to it?